### ISSUE TITLE
Whitespace character in AMRFinderPlus directory path

### ISSUE THREAD

**Issue by amridom** [ncbi/amr#23]
Thanks again for your useful tool and kind support.
As it is represented below, I am running AMRFindePlus on Ubuntu OS with an Campylobacter jejuni by NCBI accession no. NZ_LS483362.1. In below code the working directory is where "amrfinder" file located and relative path were used for parameters:
```
amir@amir-VirtualBox:~/Desktop/AMRFinder Plus/3.6.10$ ./amrfinder -n ../../NCTC11168\ NZ_LS483362.fasta --database ../databases/3.6/2020-01-22.1/ --organism Campylobacter --plus --mutation_all ../../mutation.csv --blast_bin ../../ncbiblast/bin/ --output ../../output.csv
Running ./amrfinder -n '../../NCTC11168 NZ_LS483362.fasta' --database ../databases/3.6/2020-01-22.1/ --organism Campylobacter --plus --mutation_all ../../mutation.csv --blast_bin ../../ncbiblast/bin/ --output ../../output.csv
Software directory: /home/amir/Desktop/AMRFinder Plus/3.6.10/
Database directory: ../databases/3.6/2020-01-22.1/
Software version: 3.6.10
Database version: 2020-01-22.1
AMRFinder translated nucleotide and mutation search with database ../databases/3.6/2020-01-22.1/: /home/amir/Desktop/AMRFinder Plus/databases/3.6/2020-01-22.1
Running blastx...
sh: 1: /home/amir/Desktop/AMRFinder: not found

*** ERROR ***
Loading file "/tmp/34WZNe.log"
Cannot open file "/tmp/34WZNe.log"

HOSTNAME: ?
SHELL: /bin/bash
PWD: /home/amir/Desktop/AMRFinder Plus/3.6.10
Progam name:  amrfinder
Command line: ./amrfinder -n '../../NCTC11168 NZ_LS483362.fasta' --database ../databases/3.6/2020-01-22.1/ --organism Campylobacter --plus --mutation_all ../../mutation.csv --blast_bin ../../ncbiblast/bin/ --output ../../output.csv
```
Later in following code absolute path with "\" (skip) character were used in the command line:
```
amir@amir-VirtualBox:~$ /home/amir/Desktop/AMRFinder\ Plus/3.6.10/amrfinder -n /home/amir/Desktop/NCTC11168\ NZ_LS483362.fasta --database /home/amir/Desktop/AMRFinder\ Plus/databases/3.6/2020-01-22.1/ --organism Campylobacter --plus --mutation_all /home/amir/Desktop/mutation.csv --blast_bin /home/amir/Desktop/ncbiblast/bin/ --output /home/amir/Desktop/output.csv
Running '/home/amir/Desktop/AMRFinder Plus/3.6.10/amrfinder' -n '/home/amir/Desktop/NCTC11168 NZ_LS483362.fasta' --database '/home/amir/Desktop/AMRFinder Plus/databases/3.6/2020-01-22.1/' --organism Campylobacter --plus --mutation_all /home/amir/Desktop/mutation.csv --blast_bin /home/amir/Desktop/ncbiblast/bin/ --output /home/amir/Desktop/output.csv
Software directory: /home/amir/Desktop/AMRFinder Plus/3.6.10/
Database directory: /home/amir/Desktop/AMRFinder Plus/databases/3.6/2020-01-22.1/
Software version: 3.6.10
Database version: 2020-01-22.1
AMRFinder translated nucleotide and mutation search with database /home/amir/Desktop/AMRFinder Plus/databases/3.6/2020-01-22.1/: /home/amir/Desktop/AMRFinder Plus/databases/3.6/2020-01-22.1
tail: cannot open '/home/amir/Desktop/AMRFinder' for reading: No such file or directory
tail: cannot open 'Plus/databases/3.6/2020-01-22.1//AMRProt-mutation.tab' for reading: No such file or directory
tail: cannot open '/home/amir/Desktop/AMRFinder' for reading: No such file or directory
tail: cannot open 'Plus/databases/3.6/2020-01-22.1//taxgroup.tab' for reading: No such file or directory

*** ERROR ***
Possible organisms: 

HOSTNAME: ?
SHELL: /bin/bash
PWD: /home/amir
Progam name:  amrfinder
Command line: '/home/amir/Desktop/AMRFinder Plus/3.6.10/amrfinder' -n '/home/amir/Desktop/NCTC11168 NZ_LS483362.fasta' --database '/home/amir/Desktop/AMRFinder Plus/databases/3.6/2020-01-22.1/' --organism Campylobacter --plus --mutation_all /home/amir/Desktop/mutation.csv --blast_bin /home/amir/Desktop/ncbiblast/bin/ --output /home/amir/Desktop/output.csv
```
Unfortunately, none of the above approaches worked. However by removing the blank space from the file path it works properly as shown below:
```
amir@amir-VirtualBox:~$ /home/amir/Desktop/AMRFinderPlus/3.6.10/amrfinder -n /home/amir/Desktop/NCTC11168\ NZ_LS483362.fasta --database /home/amir/Desktop/AMRFinderPlus/databases/3.6/2020-01-22.1/ --organism Campylobacter --plus --mutation_all /home/amir/Desktop/mutation.csv --blast_bin /home/amir/Desktop/ncbiblast/bin/ --output /home/amir/Desktop/output.csv
Running /home/amir/Desktop/AMRFinderPlus/3.6.10/amrfinder -n '/home/amir/Desktop/NCTC11168 NZ_LS483362.fasta' --database /home/amir/Desktop/AMRFinderPlus/databases/3.6/2020-01-22.1/ --organism Campylobacter --plus --mutation_all /home/amir/Desktop/mutation.csv --blast_bin /home/amir/Desktop/ncbiblast/bin/ --output /home/amir/Desktop/output.csv
Software directory: /home/amir/Desktop/AMRFinderPlus/3.6.10/
Database directory: /home/amir/Desktop/AMRFinderPlus/databases/3.6/2020-01-22.1/
Software version: 3.6.10
Database version: 2020-01-22.1
AMRFinder translated nucleotide and mutation search with database /home/amir/Desktop/AMRFinderPlus/databases/3.6/2020-01-22.1/: /home/amir/Desktop/AMRFinderPlus/databases/3.6/2020-01-22.1
Running blastx...
Running blastn...
AMRFinder took 180 seconds to complete
```
Since using the blank space character is inevitable in many cases, it would be appreciated if you can help me how can I find a solution for this matter. Thanks for your consideration.
Regards,
Amir

**Comment by evolarjun** [comment]
Hello again Amir,

The path handling was recently redone to some extent, and obviously some issues remain. I haven't tried it yet, but I think you've hit on the problem and I'll see if we can put together a fix for the next version of AMRFinderPlus. In the meantime hopefully you can handle it by removing spaces from the path to AMRFinderPlus.

Thanks for reporting,
Arjun

**Comment by evolarjun** [comment]
Sorry to have taken so long with the release, but this should be fixed by release 3.6.15. (https://github.com/ncbi/amr/releases/tag/amrfinder_v3.6.15).  Let us know if you have further issues.
